lerna.json

```diff
@@ -0,0 +1,12 @@
+{
+	"lerna": "2.11.0",
+	"commands": {
+		"publish": {
+			"message": "chore(release): publish %s"
```

Comment:
If the packages are going to be _independently_ released as they are currently with the _packages_ repo the `%s` here will be ignored and should be dropped here, see https://github.com/lerna/lerna#--message--m-msg

Reply:
I thought we keep one version aligned with Gutenberg release version. I need to collect all my thoughts and move to description :)
So the idea I had is that every time we publish a new version of the plugin to .org, we publish also the same version of npm packages. We will have to revisit after 5.0 mission is accomplished.

Reply:
Yes, makes sense 👍 

On the `%s` issue, did you run `lerna init` with the `--independent` flag?  If so `%s` will be ignored, I'll take a closer look tomorrow, its a pretty minor thing that we will see if it is or is not when `lerna publish` is run for th first time 😏

Reply:
`lerna init` only :)